Vechain-EasyA-Hackathon-Boston-Oct-2023/milestone-deliveries/Cypress-1.md

```diff
@@ -0,0 +1,26 @@
+
+# Milestone Delivery :mailbox:
+
+> Only the GitHub account, which is responsible for the pull request of the accepted application is allowed to submit milestones. Don't remove any of the mandatory parts presented in bold letters or as headlines!
+
+**The Ethereum address has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](../#milestone-delivery-process).**  
+
+* **PR Link:** [#207](https://github.com/vechain/grant-program/pull/207)
+
+* **Milestone Number:** Milestone 1
+
+Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which should be used for testing.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Research |[Research Presentation](https://www.canva.com/design/DAF6cfFuufM/vcdMTYG8sdO6qLO6cJ7nWA/edit)| All consumer data and research available here, includes survey respondent data + desired workflow. Slides 1-7| 
+| 2.  | Requirements |[Requirements Document](https://docs.google.com/document/d/1iFZB31XT41khzqHYUzkxCHCL0ov-M5TMTjD0i8L0lE0/edit?usp=sharing)| Clear and concise requirements for blockchain integration, product tracking, transparent sustainability metrics, and technical implementation|
+| 3.  | Outreach |[Research Presentation](https://www.canva.com/design/DAF6cfFuufM/vcdMTYG8sdO6qLO6cJ7nWA/edit)| Contacted potential investors/partners such as Uber, BLUEBIKES, MBTA to see if a partnership is possible. Slides 11-14|
+| 4.  | Designs |[Competitive Analysis + UX/UI Details](https://www.figma.com/file/xYG6HZh7j5H0acJYD8oKcr/Cypress-Competitive-Analysis?type=whiteboard&node-id=0-1&t=PGb3V4NISAxEFM49-0) | Competitive analysis performed and refined the current UI/UX system based on market research and necessary requirements set out by consumers and brands.| 
```

Comment:
The link only contains the competitive analysis, please also include the UI/UX details.

Reply:
Hello, the link is located here. https://www.figma.com/file/llEhoaZHO4GR2qSks42yjW/Cypress?type=design&node-id=26%3A726&mode=design&t=Uev1HLic5YHzOJli-1

Reply:
Please reflect the link in to the delivery table.

Reply:
Hi, updated onto delivery table and freshly committed!